ethereum/utils.py

```diff
@@ -35,6 +35,9 @@ def to_string_for_regexp(value):
         return str(value)
     unicode = unicode
 
+    def bytearray_to_bytestr(value):
+        return bytes(''.join(chr(c) for c in value))
```

Comment:
So this function is used in some places where `value` isn't necessarily a `bytearray` but rather a list of bytes (as integers).  In that case `str(value)` doesn't do the right thing.